src/views/admin/PersonManagementView.vue

```diff
@@ -46,14 +47,21 @@
       );
       const rollen: string = uebersicht?.zuordnungen.length
         ? uebersicht.zuordnungen.map((zuordnung: Zuordnung) => zuordnung.rolle).join(', ')
-        : '-';
+        : '---';
       // Choose sskDstNr if available, otherwise sskName.
       const administrationsebenen: string = uebersicht?.zuordnungen.length
         ? uebersicht.zuordnungen
             .map((zuordnung: Zuordnung) => (zuordnung.sskDstNr ? zuordnung.sskDstNr : zuordnung.sskName))
             .join(', ')
-        : '-';
-      return { ...person, rollen: rollen, administrationsebenen: administrationsebenen };
+        : '---';
+      // Check if personalnummer is null, if so, replace it with "---"
+      const personalnummer: string = person.person.personalnummer ? person.person.personalnummer : '---';
```

Comment:
```suggestion
      const personalnummer: string = person.person.personalnummer ?? '---';
```
siehe oben. Wäre cool, wenn das auch funktioniert.